### DISCUSSION TITLE
Is it possible to do vertical tabs

### DISCUSSION THREAD

**Discussion by markcarroll** [saadeghi/daisyui#2348] [Ideas / feature requests]
Is it possible to build a vertical version of the tab components?. We would like to be able to have popup dialogs that have multiple tabs on the side that would be implemented with this component.

**Comment by saadeghi** [comment]
Currently, tab doesn't have this style. Maybe we add this style in future versions.  
But you can use [menu](https://daisyui.com/components/menu/) which can be vertical or horizontal
